### DISCUSSION TITLE
Strange Analog Meter

### DISCUSSION THREAD

**Discussion by firesgc** [jomjol/AI-on-the-edge-device#1186] [Q&A] [answered]
Hello,

I have a somewhat strange analog meter (the one for 0.001) that can't be read properly. Does anyone else have such a device and know a solution for it?

Thanks!

![image](https://user-images.githubusercontent.com/19313921/196177222-3d841ac8-50b4-4272-a7f3-350398ae6a83.png)

**Comment by jomjol** [comment, marked as answer]
You would need to create a set of training data to hand this.

OR: you use only up to the x0.01 with extended resolution and ignore the x0,001 and x.0001. In this case you still have 1l resolution.

**Reply by firesgc** [reply]
I will choose the second approach. 1l resolution is good enough :)
Thanks for the support!
